test/modules/post/test/mssql.rb

```diff
@@ -0,0 +1,78 @@
+require 'rex/post/meterpreter/extensions/stdapi/command_ids'
+require 'rex'
+
+lib = File.join(Msf::Config.install_root, "test", "lib")
+$LOAD_PATH.push(lib) unless $LOAD_PATH.include?(lib)
+require 'module_test'
+
+class MetasploitModule < Msf::Post
+
+  include Msf::ModuleTest::PostTest
+  include Msf::ModuleTest::PostTestFileSystem
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Testing MSSQL sessions work',
+        'Description' => %q{ This module will test the mssql sessions work },
+        'License' => MSF_LICENSE,
+        'Author' => [ 'zachgoldman'],
+        'Platform' => all_platforms,
```

Comment:
You may be able to use either all or an empty string here instead.

https://github.com/zgoldman-r7/metasploit-framework/blob/a1613d6070cd9a481c813b8abc12c2acf9b89e3d/lib/msf/core/module/platform.rb#L14